docs/Community/sig-testing/README.md

```diff
@@ -0,0 +1,35 @@
+Testing and CI/CD Special Interest Group
+========================================
+
+All things testing within the OE SDK project.
+
+This SIG is specifically interested in making it easier for the OE SDK community
+to run and contribute tests, as well as integrate downstream testing, to ensure
+the OE SDK is stable across a wide range of hardware and use-cases.
+
+This SIG is responsible for defining the overall approach to testing OE SDK
+itself, performing third-party testing and validation, and reviewing PRs that
+affect *how* testing is performed broadly.
+
+This SIG may also weigh in on specific PRs, though generally speaking, other
+SIGs should be responsible for test coverage within their respective domains.
+
+Meetings
+--------
+
+Regular SIG-Testing meeting: Weekly on [Tuesdays at 17:30 PT (Pacific Time)](https://us04web.zoom.us/j/119965269?pwd=bW1GZWJXdWlOb3BldFN5aFFCSTNNQT09 ) [(convert to your local timezone)](https://www.thetimezoneconverter.com/?t=17:30&tz=PT%20%28Pacific%20Time%29)
+
+[Meeting Agenda and Notes](https://hackmd.io/@aeva/oesdk-sig-testing)
+
+SIG leads
+---------
+
+TBD
+
+Meeting Chair
+-------------
+
+TBD
+
+Charter
+-------
```

Comment:
should there be a TBD under this line?

Reply:
Probably. But this is just a copy-pasta of a template, to demonstrate what we might put here.

I'd like to see these files filled in through community discussion  :)